jackson-jq/src/main/java/net/thisptr/jackson/jq/path/ObjectFieldPath.java

```diff
@@ -66,8 +66,9 @@ private static JsonNode mutate(JsonNode in, final String key, final Mutation mut
 				final Entry<String, JsonNode> entry = iter.next();
 				newobj.set(entry.getKey(), entry.getValue());
 			}
-			final JsonNode newval = mutation.apply(newobj.get(key));
-			if (newval != null)
+			JsonNode newval = newobj.get(key);
+			newval = mutation.apply(newval != null ? newval : NullNode.getInstance());
+			if (newval != null && !newval.isNull())
```

Comment:
JSON `null` (NullNode) is a valid value for object fields, so `&& !newval.isNull()` is unnecessary.
```
$ jq -c '.foo += null' <<< '{}'
{"foo":null}
```
Other than that, the patch looks good :+1:

Reply:
If I remove the isNull, a lot of test will fail (acutally it took me a while to figure out that)